### DISCUSSION TITLE
Can we add CSS to the responsive image html?

### DISCUSSION THREAD

**Discussion by diazsasak** [spatie/laravel-medialibrary#2702] [Q&A]
Hi, 

can we add css to the generated <img> tag of the responsive image?

![image](https://user-images.githubusercontent.com/7880137/146292597-58b9bb22-7faf-4e56-9b74-792f9011a2bd.png)

Regards

**Comment by systemsolutionweb** [comment]
Yes, you can
only if you make a PR for that

Regards

**Comment by erikn69** [comment]
Maybe try 
```php
{{ $yourModel->getFirstMedia()->img('', ['class' => 'YOUR_CLASS']) }}
```
Or read the domuentation [customizing-the-rendered-html](https://github.com/spatie/laravel-medialibrary/blob/c0d9bcdfdd954523f817295d2fde73401e1a8e91/docs/responsive-images/customizing-the-rendered-html.md), why nobody reads?
